### PR TITLE
feat: filter without close dropdown (#28652)

### PR DESCRIPTION
**Pull request by alekye**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
[#28652]

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Execute Table custom filter without close drop down menu.|
| 🇨🇳 Chinese |Table 组件自定义筛选器，执行筛选时不关闭筛选菜单。|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/table/demo/custom-filter-panel.md](https://github.com/alekye/ant-design/blob/customfilter/components/table/demo/custom-filter-panel.md)

### CODE REVIEW

File: components/table/demo/custom-filter-panel.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ title:
 
 通过 `filterDropdown` 自定义的列筛选功能，并实现一个搜索列的示例。
 
+给函数 `confirm` 添加 `boolean` 类型参数 `closeDropdown`，是否关闭筛选菜单，默认为 `true`，关闭。
```
Comment (afc163) [inline components/table/demo/custom-filter-panel.md:12]:
```suggestion
给函数 `confirm` 添加 `boolean` 类型参数 `closeDropdown`，是否关闭筛选菜单，默认为 `true`。
```

File: components/table/demo/custom-filter-panel.md [hunk 2]
```diff
@@ -77,6 +79,19 @@ class App extends React.Component {
           <Button onClick={() => this.handleReset(clearFilters)} size="small" style={{ width: 90 }}>
             Reset
           </Button>
+          <Button
+            type="link"
+            size="small"
+            onClick={() => {
+              confirm(false);
```
Comment (afc163) [inline components/table/demo/custom-filter-panel.md:86]:
```suggestion
              confirm({ closeDropdown: false });
```

Comment (alekye) [reply components/table/demo/custom-filter-panel.md]: 已修改

**Review by afc163 — APPROVED** [review]

File: components/table/demo/custom-filter-panel.md [hunk 3]
```diff
@@ -77,6 +79,20 @@ class App extends React.Component {
           <Button onClick={() => this.handleReset(clearFilters)} size="small" style={{ width: 90 }}>
             Reset
           </Button>
+          <Button
+            type="link"
+            size="small"
+            onClick={() => {
+              confirm({ closeDropdown: false });
+              console.log('1234');
```
Comment (afc163) [inline components/table/demo/custom-filter-panel.md:87]: 这个忘记删了。

**Comment by alekye** [comment]
@afc163 ,请问这个PR怎么没有merge到master

File: components/table/demo/custom-filter-panel.md (hunk 3 above)
Comment (alekye) [reply components/table/demo/custom-filter-panel.md:87]: 需要再提交一次吗

Comment (yoyo837) [reply components/table/demo/custom-filter-panel.md:87]: 发新PR

**Comment by yoyo837** [comment]
> @afc163 ,请问这个PR怎么没有merge到master

我本想看一下为什么，但是奈何github同步不下来 😂

File: components/table/demo/custom-filter-panel.md (hunk 3 above)
Comment (alekye) [reply components/table/demo/custom-filter-panel.md:87]: 发了

**Comment by yoyo837** [comment]
4.10.0的minor版本是7天前，你这个是6天前，得等下个minor了，也就是月底的4.11.0.

**Comment by alekye** [comment]
好的，明白了，谢谢

**Pull request merged**